recipes/r-spscomps/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 build:
   merge_build_host: True  # [win]
   number: 0
-  skip: true  # [r_base < 4]
+  skip: true  # [r_base < '4']
```

Comment:
Yeah, what I usually use is `[r_base == "3.6"]`; what you use above should work.

Reply:
Maybe the double quotes helps? I'll give it a try

Reply:
It may the ´<´ actually, not sure that ever worked for me.

Reply:
I guess that's the trick